src/js/modules/infragistics.ui.editors.js

```diff
@@ -7990,6 +7990,14 @@
 			}
 			return date;
 		},
+		_setNewDateMidnight: function() {
+			var date = new Date();
+			date = this._setDateField("hours", date, 0);
+			date = this._setDateField("minutes", date, 0);
+			date = this._setDateField("seconds", date, 0);
+			date = this._setDateField("milliseconds", date, 0);
```

Comment:
Since _setDateFiled actually sets the value on date, there's no point assigning. Very minor.